### PR TITLE
Option to show/hide footer statistics

### PR DESCRIPTION
**Pull request by brownl**
This addresses issue #624.

### CODE REVIEW

File: templates/footer.inc.php [hunk 1]
```diff
@@ -19,6 +19,11 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  *
  */
+
+$ampache_path = dirname(__FILE__);
+$prefix = realpath($ampache_path . "/../");
+require_once $prefix . '/lib/init.php';
```
Comment (Afterster) [inline templates/footer.inc.php]: Why do you have these lines in footer.inc.php? Does this was for test and get committed by error?

**Comment by brownl** [comment]
without them I get errors about AmpConfig not being defined. Or am I doing something wrong?

**Comment by Afterster** [comment]
I don't think you get errors without these lines, are you sure?
Could you also clean your commits now for the PR? Several commits with revert inside the PR itself is too much noise :) Thanks.

**Comment by brownl** [comment]
Okay, opened #703. Thanks.

**Pull request closed without merging**